homeassistant/components/google_generative_ai_conversation/__init__.py

```diff
@@ -139,7 +139,17 @@ async def async_setup_entry(
     """Set up Google Generative AI Conversation from a config entry."""
 
     try:
-        client = genai.Client(api_key=entry.data[CONF_API_KEY])
```

Comment:
Can you replace this with: `client = await hass.async_add_executor_job(Client, api_key=entry.data[CONF_API_KEY])`?

Reply:
That did not work, it complains about unexpected key word argument.